src/pages/careers/create.jsx

```diff
@@ -0,0 +1,150 @@
+import { useState } from "react";
+import Button from "@/components/ui/Button";
+import PageIntro from "@/components/PageIntro";
+import { withTranslation } from "next-i18next";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { useRouter } from "next/navigation";
+import addDocument from "@/firebase/addData";
+import Input from "@/components/ui/Input";
+
+const Add_Career = ({ t }) => {
+    const [enTitle, SetEnTitle] = useState("");
+    const [arTitle, setArTitle] = useState("");
+
+    const [enDescrption, setEnDescrption] = useState("");
+    const [arDescrption, setArDescrption] = useState("");
+
+    const [enSpecialization, setEnSpecialization] = useState("");
+    const [arSpecialization, setArSpecialization] = useState("");
+
+    const [enOccupation, setEnOccupation] = useState("");
+    const [arOccupation, setArOccupation] = useState("");
```

Comment:
- I think it will be better to use all the string vars in the state with one state.
- Create a one handler method to handle the change in that state.

Reply:
I put all them together in one Use State ,Done